content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst

```diff
@@ -0,0 +1,174 @@
+with set pricing. To view and configure blanket orders, head to
+:menuselection:`Purchase app --> Orders --> Purchase agreements`.
+
+:guilabel:`Currency` can be changed if purchasing products from a vendor in another country.
+To change the main currency or enable additional currencies, head to
+:menuselection:`Configuration --> Settings --> Currencies`.
+
+:guilabel:`Deliver to` provides options for the order shipment. Select the receiving warehouse
+address here, or select :guilabel:`Dropship` to indicate that this order is to be shipped
+directly to the end customer. When :guilabel:`Dropship` is selected, the
+:guilabel:`Dropship address` field is enabled. Contact names will populate here from the
+Contacts app.
+
+:guilabel:`Expected Arrival` is automatically calculated based on the Order Deadline and Vendor Lead
+Time. Click the checkbox for :guilabel:`Ask confirmation` to ask for signage at delivery.
+
+In the products tab, add the products to be ordered. Select :guilabel:`Add a product` and start
+typing in the product name to select the item from the drop-down menu. :guilabel:`Catalog` can also
+be selected to navigate to a products menu under the chosen vendor. From here, products can be
+added to the cart. If ordering a product that has not been configured, type in the product name,
+select :guilabel:`Product Name` and manually add the unit price.
+
+.. note::
+  To make adjustments to products and prices, access the product form by clicking the
+  :icon:`oi-arrow-right` :guilabel:`(right arrow)` icon that becomes available upon hovering over
+  the product name.
+
+Send |RFQ|
+----------
+
+There are a few options on how to send an |RFQ| to vendors:
+
+:guilabel:`Send by Email` automatically loads an RFQ template email to send to the vendor’s email
+address (configured in the **Contacts** app). After crafting the desired message, click
+:guilabel:`Send`. Once the email has been sent, the |RFQ| moves to the :guilabel:`RFQ Sent` stage.
+
+:guilabel:`Print RFQ` downloads a PDF version of the |RFQ|.
+
+Confirm order
+-------------
+
+Clicking :guilabel:`Confirm Order` directly transforms the |RFQ| into an active order.
+
+.. tip::
+   Odoo tracks communications on each order through the chatter screen on the right. This shows
+   the emails sent between the user and the contact, as well as any internal notes and activities.
+   Messages, notes, and activities can also be logged on chatter.
+
+Once an |RFQ| is confirmed, it creates a |PO|. Products have been ordered!
+
+On the new |PO|, the :guilabel:`Order Deadline` turns into the :guilabel:`Confirmation Date`, which
+displays the date and time the user confirmed the order.
+
+Depending on the user's chosen configuration in the *Purchase* app settings, a *vendor bill* is
+created once products have been ordered or received. For more information on managing vendor bills,
+refer to :doc:`this documentation.<manage>`
```

Comment:
Minor nitpick-- we typically have a space between link text ("this documentation"), and the link itself ("manage")
```suggestion
refer to :doc:`this documentation <manage>`.
```